core/controllers/subtopic_viewer.py

```diff
@@ -26,6 +26,26 @@
 class SubtopicViewerPage(base.BaseHandler):
     """Renders the subtopic viewer page."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'classroom_url_fragment': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'topic_url_fragment': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'subtopic_url_fragment': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
```

Comment:
Are there more rules to these fragments?

Reply:
Hey @vojtechjelinek 
Kindly tell me how to check that.

Reply:
because in the main.py and feconf.py I am not finding more than them.

Reply:
Hey @vojtechjelinek 
There are no regex for 'subtopic_url_fragment' as for 'topic_url_fragment & classroom_url_fragment' in constants.ts file.
What to do next?

Reply:
Isn't there a max limit validator? If so, you can use the relevant constants in `android_validation_constants` for these.